Frontend/CO.CDP.OrganisationApp/Pages/Organisation/SupplierToBuyerDevolvedRegulation.cshtml

```diff
@@ -0,0 +1,60 @@
+@page "/organisation/{id}/supplier-to-buyer-devolved-regulation"
+@using CO.CDP.OrganisationApp.Constants
+
+@model CO.CDP.OrganisationApp.Pages.Organisation.SupplierToBuyerDevolvedRegulationModel
+@{
+    ViewData["Title"] = "Do devolved regulations apply to your organisation?";
+    var devolvedHasError = ((TagBuilder)Html.ValidationMessageFor(m => m.Devolved)).HasInnerHtml;
+
+    var backLink = $"/organisation/{@Model.Id}/supplier-to-buyer-devolved-regulation";
+
+}
+
+<a href="@backLink" class="govuk-back-link">
+    Back
+</a>
+
+<main class="govuk-main-wrapper" id="main-content">
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds govuk-body">
+            <partial name="_ErrorSummary" model="@ModelState" />
+            <form method="post">
+                <div class="govuk-form-group @(devolvedHasError ? "govuk-form-group--error" : "")">
+                    <fieldset class="govuk-fieldset">
+                        <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+                            <h1 class="govuk-fieldset__heading">
+                                @ViewData["Title"]
+                            </h1>
+                        </legend>
+                        @if (devolvedHasError)
+                        {
+                            <p id="devolved-error" class="govuk-error-message">
+                                <span class="govuk-visually-hidden">Error:</span> @Html.ValidationMessageFor(m => m.Devolved)
+                            </p>
+                        }
+
+                        <div class="govuk-radios" data-module="govuk-radios">
+                            <div class="govuk-radios__item">
+                                <input class="govuk-radios__input" id="devolved-yes" name="Devolved" type="radio" value="true" @(Model.Devolved == true ? "checked" : "")>
+                                <label class="govuk-label govuk-radios__label" for="devolved-yes">
+                                    Yes
+                                </label>
+                            </div>
+                            <div class="govuk-radios__item">
+                                <input class="govuk-radios__input" id="devolved-na" name="Devolved" type="radio" value="false" @(Model.Devolved == false ? "checked" : "")>
+                                <label class="govuk-label govuk-radios__label" for="devolved-na">
+                                    No
+                                </label>
+                            </div>
+                        </div>
+                    </fieldset>
+                </div>
+                <a href="/registration/buyer-select-devolved-regulations">
+                    <govuk-button>
+                        Continue
+                    </govuk-button>
+                </a>
```

Comment:
assuming you want this form to post somewhere, this `<a>` tag wrapping the button needs to go

Reply:
This <a> has been removed now.